host-spec/c03-transition.tm

```diff
@@ -7,37 +7,39 @@
 <\body>
   <chapter|State Transition><label|chap-state-transit>
 
-  Like any transaction-based transition system, Polkadot state changes via
-  executing an ordered set of instructions. These instructions are known as
-  <em|extrinsics>. In Polkadot, the execution logic of the state-transition
-  function is encapsulated in Runtime as defined in Definition
-  <reference|defn-state-machine>. Runtime is presented as a Wasm blob in
-  order to be easily upgradable. Nonetheless, the Polkadot Host needs to be
-  in constant interaction with Runtime. The detail of such interaction is
-  further described in Section <reference|sect-entries-into-runtime>.
+  Like any transaction-based transition system, Polkadot's state is changed
+  by executing an ordered set of instructions. These instructions are known
+  as <em|extrinsics>. In Polkadot, the execution logic of the
+  state-transition function is encapsulated in a Runtime as defined in
+  Definition <reference|defn-state-machine>. For easy upgradability this
```

Comment:
> For **easier** upgradability **and in order to avoid hard forks in the Polkadot network,** this ...

We'd probably need to define "hard forks", then. Up to you.

Reply:
I would avoid that for now.